lib/paperclip/io_adapters/identity_adapter.rb

```diff
@@ -1,12 +1,12 @@
 module Paperclip
   class IdentityAdapter < AbstractAdapter
-    def new(adapter)
-      adapter
```

Comment:
Perhaps @jyurek could shine some light on what this adapter is supposed to do because it confused me as well.